charabia/src/normalizer/ae_oe_normalizer.rs.rs

```diff
@@ -0,0 +1,162 @@
+// Import `CharNormalizer` trait.
+use super::{ CharNormalizer, CharOrStr };
+use crate::Token;
+
+// Make a small documentation of the specialized Normalizer like below.
+/// <Script/Language> specialized [`Normalizer`].
+///
+/// This Normalizer uses [`<UsedLibraryToNormalize>`] internally to normalize the provided token.
+/// <OptionalAdditionnalExplanations>
```

Comment:
Can you please modify the documentation in order to explain the implementation, please?